### PR TITLE
Add `$default` parameter to `\Parable\GetSet\Base::get()`

### PR DESCRIPTION
**Pull request by dmvdbrugge**
to return when `$key` not found. Default `null` for BC.

### CODE REVIEW

File: src/GetSet/Base.php [hunk 1]
```diff
@@ -53,22 +53,23 @@ public function getAll()
     }
 
     /**
-     * Return specific value by key if resource set.
+     * Return specific value by key if resource set. If not found, return default.
      *
-     * $getSet->get("one.two.three", "value") would return $resource["one"]["two"]["three"];
```
Comment (devvoh) [inline src/GetSet/Base.php]: And this takes care of this copy/paste snafu as well ;)

**Comment by devvoh** [comment]
Man, Travis has been slow all day :/

**Comment by dmvdbrugge** [comment]
At least StyleCI is fast 😉

**Comment by devvoh** [comment]
Yay. Go StyleCI -_-

**Pull request merged**